cmd/internal/cli/pull.go

```diff
@@ -217,7 +230,8 @@ func pullRun(cmd *cobra.Command, args []string) {
 			sylog.Fatalf("While pulling library image: %v", err)
 		}
 	case ShubProtocol:
-		err := shub.DownloadImage(name, args[i], force, noHTTPS)
+		// TODO:  <--
```

Comment:
Same here

Reply:
Ohh, this is a reminder about the shub caching. I will remove it.